cgi-bin/LJ/CleanHTML.pm

```diff
@@ -617,7 +628,14 @@ TOKEN:
                     : exists $attr->{comm} ? $attr->{comm}
                     :                        undef;
 
-                $newdata .= user_link_html( $user, $attr->{site}, $opts );
+                if ( $opencount{'a'} ) {
```

Comment:
nit:

Wouldn't it have been semantically the same and cleaner to just do,

```perl
$opts->{delink_user_links} = $opencount{'a'} ? 1 : 0;
$newdata .= user_link_html( $user, $attr->{site}, $opts );
```

Since, in the first path, you're unconditionally setting it -- but not in the second path -- seems like we should just always set it appropriately (and make sure it gets a default initizliation state in opts at the beginning of the sub?).